src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -182,13 +182,11 @@ public override void VisitNamedType(INamedTypeSymbol symbol)
             AddNullableAnnotations(symbol);
 
             if ((format.CompilerInternalOptions & SymbolDisplayCompilerInternalOptions.IncludeContainingFileForFileTypes) != 0
-                // PROTOTYPE(ft): public API?
```

Comment:
>// PROTOTYPE(ft): public API? [](http://example.com/codeflow?start=16&length=29)

Does the PROTOTYPE comment no longer apply?

Reply:
Moved the question to the test plan #60819:

> * public API: should we have `bool INamedTypeSymbol.IsFile`?
> * public API: should we have public SymbolDisplayOptions to show the associated file of a named type? e.g. `NS.MyClass@MyFile`.